_includes/release_data.html

```diff
@@ -10,13 +10,13 @@
 }}
 {%- capture releaseChangelog -%}
 <ul>
-    <li>Dropping support for Android KitKat</li>
-    <li>Download videos directly through the long-press menu in lists</li>
-    <li>Share local playlists as list of video URLs</li>
-    <li>Add option to hide future videos (a.k.a premieres)</li>
+    <li>Comment replies as well as support for searching for playlists and channels on PeerTube</li>
```

Comment:
Users may think that the app got comment replies support only for PeerTube, where that's an extractor-only addition. We should remove this part to avoid any confusion.

Reply:
oh, same for searching, that is also extractor only?

Reply:
No, the additions to PeerTube searches work in the app.